cmd/kubelet/app/server.go

```diff
@@ -155,9 +154,6 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			// short-circuit on verflag
 			verflag.PrintAndExitIfRequested()
 
-			// log args (separate lines, so we don't overflow line-length limits in logging infrastructure)
-			glog.V(2).Infof("kubelet flags: %s", strings.Join(args, "\n"))
```

Comment:
Kubelet intentionally avoids using the global flagset, and as a result the flag logging should be done here. As @liggitt said below, `os.Args` is not the same as flags.

Reply:
ack. i'll add this back